horovod/run/run.py

```diff
@@ -609,6 +612,12 @@ def parse_args():
                                   help='Run Horovod using the MPI controller. This will '
                                        'be the default if Horovod was built with MPI support.')
 
+    parser.add_argument('--btl_tcp_if_include', action='store', dest='btl_tcp_if_include',
```

Comment:
There is already a parameter `--network-interface` that does something very similar.  So I would suggest reusing it.  However, there are two things we'd need to change to make it work the way this PR does:

1. `--network-interface` should accept a comma-separate list of NICs
2. Existing behavior still checks that `--network-interface` is common among all hosts, so we should skip this step (as you do here).

Reply:
Thanks for pointing it out.